examples/kubernetes/generator/README.md

```diff
@@ -0,0 +1,54 @@
+Ceph Kubernetes Secret Generation
+=================================
+
+This script will generate ceph keyrings and configs as Kubernetes secrets.
+
+Sigil is required for template handling and must be installed in system PATH. Instructions can be found here: https://github.com/gliderlabs/sigil
```

Comment:
it looks sigil is only needed to replace key here, in such case, why not just use sed and remove sigil dependency?